app/models/tweet_prep.rb

```diff
@@ -0,0 +1,29 @@
+class TweetPrep
+  def self.bounty_hashtags(bounty)
+    QueryMarks.new.legible_mark_vector(bounty.mark_vector).sort_by{|a, b| -b}.map{|a, b| a}.take(4)
```

Comment:
Unused block argument - `a`. If it's necessary, use `_` or `_a` as an argument name to indicate that it won't be used.<br>Unused block argument - `b`. If it's necessary, use `_` or `_b` as an argument name to indicate that it won't be used.<br>Line is too long. [99/80]<br>Space missing to the left of {.<br>Space between { and | missing.<br>Space missing inside }.